src/lib.rs

```diff
@@ -350,17 +350,17 @@ impl<L: AnalysisLoader> AnalysisHost<L> {
         })
     }
 
-    pub fn name_defs(&self, name: &str) -> AResult<Vec<Def>> {
+    pub fn matching_defs(&self, query: &str) -> AResult<Vec<Def>> {
```

Comment:
I would prefer `substr` to `query` as an arg. Could you add a comment explaining how this matches (substring, case-insensitive) please?